src/components/SupportRequestBaseModal.js

```diff
@@ -0,0 +1,78 @@
+import { Component } from 'react';
+import { button, div, h, span, hh } from 'react-hyperscript-helpers';
+import Modal from 'react-modal';
+import './BaseModal.css';
```

Comment:
I would leave this out and instead define any styles you need as js objects here in this component. It's a lot easier to refactor and find dead code this way. CSS in css files is very difficult to maintain over time.